### PR TITLE
rtmidi: new recipe

### PR DESCRIPTION
**Pull request by Cacodemon345**
This ports the popular RtMidi library to Haiku.

### CODE REVIEW

File: media-libs/rtmidi/rtmidi-5.0.0.recipe [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	rtmidi${secondaryArchSuffix}_devel = $portVersion
+	devel:librtmidi$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	haiku$secondaryArchSuffix
+	lib:librtmidi$secondaryArchSuffix == $portVersion
```
Comment (korli) [inline media-libs/rtmidi/rtmidi-5.0.0.recipe:40]:
```suggestion
	rtmidi$secondaryArchSuffix == $portVersion
```

**Comment by korli** [comment]
one comment in REQUIRES_devel, otherwise looks good.

**Comment by korli** [comment]
@Cacodemon345  build failed on x86: https://build.haiku-os.org/buildmaster/master/x86_gcc2/logviewer.html?buildruns/2492/builds/77654.log

**Comment by Cacodemon345** [comment]
The GCC2 compiler isn't supported since it requires C++11.

**Comment by pulkomandy** [comment]
There is  a mix of int32 and int types.
On 64bit systems, int32 is defined as int. But on 32bit it is defined as long int.

**Comment by Cacodemon345** [comment]
https://github.com/haikuports/haikuports/pull/6711
@pulkomandy @korli 

Pardon me for not testing the patch with 32-bit since I don't have extra media handy right now for that.

**Comment by pulkomandy** [comment]
That's ok, but in that case you can mark it untested, for example:

    SECONDARY_ARCHITECTURES = "?x86"

Then someone can test and enable it later.

**Comment by Cacodemon345** [comment]
I will do so next time, thanks.

**Pull request merged**